src/systems/a320_systems/src/pneumatic.rs

```diff
@@ -3119,6 +3255,378 @@ mod tests {
         assert!(!test_bed.precooler_inlet_pressure(2).is_nan());
     }
 
+    mod wing_anti_ice {
```

Comment:
I'm honestly not sure if it would not be better to have the tests in wing_anti_ice.rs. But I also understand why you put it here.

Reply:
Initially they were under wing_anti_ice, but I repeated the PneumaticTestBed code. So I thought to move them to Pneumatic. Another reason is that WAI is a user/subsystem of the big pneumatic system

Reply:
I see. I think it's fine to be there.